app/coffeescript/components/ui/markers.coffee

```diff
@@ -64,6 +62,16 @@ define [
       @updateListingsCount()
       @trigger 'uiSetMarkerInfoWindow'
 
+    @clearAllMarkers = ->
```

Comment:
What do you think about moving `markerClusterer.unbindMarkers` here? That way it's applied w/o the clusterer.